features/pagination.feature

```diff
@@ -52,17 +52,17 @@ Background: Logged in
 
```

Comment:
Already included in previous PR, but to pass CI, we have to include modified tests.

Please ignore this file.